cli.ts

```diff
@@ -176,6 +177,28 @@ const commonCLI = (env: EnvVars, args: DenoArgs, i18n: i18n.t) =>
 					forkCatch(console.error)(console.error)(console.log),
 				),
 		)
+		.command(
+			'optin',
+			i18n.__('optInDesc'),
+			() => {},
+			() =>
+				pipe(
+					optInAnalytics(),
+					map(rv => rv === true ? "You've successfully opt in for anonymous usage reporting" : ''),
+					forkCatch(console.error)(console.error)(console.log),
```

Comment:
Should we use displayError here instead of console.error?

Reply:
@mweichert Possibly have this as part of a general refactor PR? I see it in other places as well

Reply:
Ok, sounds good.